client/public/js/whiteboard.js

```diff
@@ -1,11 +1,7 @@
 let ctx, color = '#000';
 let canvasState = true;
 const moveCanvas = function() {
-  if (canvasState) {
-    canvasState = false;
-  } else {
-    canvasState = true;
-  }
+  canvasState ^= true;
```

Comment:
I'm not sure if you realize this, but it actually converts `canvasState` from a boolean to an integer (`1` or `0`)